internal/tools/aws/secrets_manager.go

```diff
@@ -227,29 +227,29 @@ func (a *Client) secretsManagerGetRDSSecret(awsID string, logger log.FieldLogger
 }
 
 func (a *Client) secretsManagerEnsureIAMAccessKeySecretDeleted(awsID string, logger log.FieldLogger) error {
-	return a.secretsManagerEnsureSecretDeleted(IAMSecretName(awsID), logger)
+	return a.secretsManagerEnsureSecretDeleted(IAMSecretName(awsID), false, logger)
 }
 
 func (a *Client) secretsManagerEnsureRDSSecretDeleted(awsID string, logger log.FieldLogger) error {
-	return a.secretsManagerEnsureSecretDeleted(RDSSecretName(awsID), logger)
+	return a.secretsManagerEnsureSecretDeleted(RDSSecretName(awsID), false, logger)
 }
 
-func (a *Client) secretsManagerEnsureSecretDeleted(secretName string, logger log.FieldLogger) error {
-	_, err := a.Service().secretsManager.DeleteSecret(&secretsmanager.DeleteSecretInput{
-		SecretId: aws.String(secretName),
+func (a *Client) secretsManagerEnsureSecretDeleted(secretName string, force bool, logger log.FieldLogger) error {
+	response, err := a.Service().secretsManager.DeleteSecret(&secretsmanager.DeleteSecretInput{
```

Comment:
tiny nit: I like to put the type on its own line in cases like this where the method call and type name of the argument are both quite long, just to aid in readability when the column is narrow like in GitHub diffs